backend/service-graphql/src/main/appengine/app.yaml

```diff
@@ -0,0 +1,11 @@
+service: graphql
+runtime: java11
+instance_class: F2
```

Comment:
Instances need 512MB of RAM now... Not sure why. Maybe the DataStore client libraries...